### PR TITLE
New Presence: SiriusXM

### LINKED ISSUE #522: SiriusXM | player.siriusxm.com
**What's the name of the service?**
Ex. player.siriusxm.com | SiriusXM

**What should the Presence display?**
Information about the current song and artist. (The station info isn't required, but would be nice.)
**Why do you want support for this service?**
SiriusXM is a great online radio streaming platform and I would love to have support for it, as it is one of my daily drivers for music. It would be nice to be able to share what I am listening to on it with my friends on Discord.

**Are you able to provide an image for the service? (512x512 minimum)**
SiriusXM

### PR DESCRIPTION
**Pull request by SnoweyFoxx**
Resolves #522 
image
image
image
image
image

### CODE REVIEW

File: SiriusXM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "id": "604182077088202784"
+    },
+    "url": "player.siriusxm.com",
+    "description": {
+      "en": "SiriusXM is like nothing you've ever heard before — it's a whole new world of radio. Imagine channels and channels of anything you want to listen to."
+    },
+    "service": "SiriusXM",
+    "version": "1.0",
+    "logo": "https://user-images.githubusercontent.com/56901798/67913602-a1672d00-fb5b-11e9-94d0-f18297af306b.png",
+    "thumbnail": "https://user-images.githubusercontent.com/56901798/67913602-a1672d00-fb5b-11e9-94d0-f18297af306b.png",
```
Comment (frozaum) [inline SiriusXM/dist/metadata.json]: Please re-upload those images to a proper image hosting service and change the thumbnail to a promotional image or a screenshot of the website.

File: SiriusXM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "color": "#0081FF",
+    "tags": [
```
Comment (frozaum) [inline SiriusXM/dist/metadata.json]: little thing, not necessary: i would recommend add the 'radio' tag and some other ones, if you want

File: SiriusXM/dist/metadata.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
   "service": "SiriusXM",
   "version": "1.0",
   "logo": "https://imgur.com/WKHpLxp.jpg",
-  "thumbnail": "https://imgur.com/WKHpLxp.jpg",
+  "thumbnail": "https://imgur.com/a/lTYYEyi.png",
```
Comment (frozaum) [inline SiriusXM/dist/metadata.json:13]: Please provide a direct link of the image.

File: SiriusXM/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "author": {
+    "name": "Alexx",
+    "id": "604182077088202784"
+  },
+  "url": "player.siriusxm.com",
+  "description": {
+    "en": "SiriusXM is like nothing you've ever heard before — it's a whole new world of radio. Imagine channels and channels of anything you want to listen to."
+  },
+  "service": "SiriusXM",
+  "version": "1.0",
```
Comment (Bas950) [inline SiriusXM/dist/metadata.json:11]: Version 1.0.0

File: SiriusXM/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  "logo": "https://imgur.com/WKHpLxp.jpg",
+  "thumbnail": "https://imgur.com/lTYYEyi.png",
```
Comment (Bas950) [inline SiriusXM/dist/metadata.json:13]: should be i.imgur linsk

Comment (doomlerd) [reply SiriusXM/dist/metadata.json:13]: + the thumbnail was deleted

**Pull request merged**